extensions/wikia/AdEngine/js/spec/lookup/prebid/adapters/indexExchange.spec.js

```diff
@@ -0,0 +1,49 @@
+describe('ext.wikia.adEngine.lookup.prebid.adapters.indexExchange', function () {
+	'use strict';
+
+	function getIndexExchange() {
+		return modules['ext.wikia.adEngine.lookup.prebid.adapters.indexExchange'](
+			mocks.geo,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the declaration of "mocks" before this usage. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)